styles.css

```diff
@@ -1 +1,68 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+.grid-container {
+    background-color: antiquewhite;
+    display: grid;
+    row-gap: 16px;
+    column-gap: 16px;
+
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 50px 240px 100px auto 50px;
+    grid-template-areas:
+    "header aside-first aside-first"
+    "main aside-first aside-first"
+    "main aside-third aside-second"
+    "main aside-third ."
+    "footer footer footer";
+}
+
+@media screen and (max-width: 576px) {
+    .grid-container {
+        grid-template-columns: 1fr;
+        grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
+        grid-template-areas:
+        "header"
+        "main"
+        "aside-first"
+        "aside-second"
+        "aside-third"
+        "."
+        "footer";
+    }
+}
+
+.grid-item {
+    padding: 16px;
+    color: antiquewhite;
+    background-color: #c41e3d;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    border-style: solid;
```

Comment:
Deze zou je kunnen combineren tot een 'shorthand' zoals ook in de uitwerking waar je de breedte, stijl en kleur in een regel combineert: border: 2px solid #7d1128;